src/main/java/de/ait/secondlife/security/config/SecurityConfig.java

```diff
@@ -50,6 +50,11 @@ public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Excepti
                         .requestMatchers(HttpMethod.PUT, "/v1/categories/{category-id}").hasRole("ADMIN")
                         .requestMatchers(HttpMethod.DELETE, "/v1/categories/{category-id}").hasRole("ADMIN")
                         .requestMatchers(HttpMethod.PATCH, "/v1/categories/{category-id}/set-active").hasRole("ADMIN")
+                        .requestMatchers(HttpMethod.GET, "/v1/offer/**").permitAll()
+                        .requestMatchers(HttpMethod.GET, "/v1/offer/user/**").authenticated()
```

Comment:
let's not allow the possibility of slipping through with a flexible value **. I suggest hard fixing the endpoints:
                        .requestMatchers(HttpMethod.GET, "/v1/offers/user/{id}").authenticated()
                        .requestMatchers(HttpMethod.POST, "/v1/offers").authenticated()
                        .requestMatchers(HttpMethod.PUT, "/v1/offers").authenticated()
                        .requestMatchers(HttpMethod.DELETE, "/v1/offers/{id}").authenticated()